### PR TITLE
Fix Bug 1174423 - Disable font-variant: small-caps for ga-IE, evaluate exclusion of other locales

### PR DESCRIPTION
**Pull request by kyoshino**
We still have to figure out locales need to be updated. So this is WIP. DO NOT MERGE.

### CODE REVIEW

**Comment by EmiXLabs** [comment]
Also need for 'az' locale

**Comment by flodolo** [comment]
Emin, do you need to disable both? Please leave a comment in the bug

**Comment by EmiXLabs** [comment]
@flodolo looks like this pull request will not be merged as its not used in mozilla.org and problem is only for Chrome. So it can be passed.

**Comment by flodolo** [comment]
mozilla.org is not used only with Firefox. It will be merged as soon as it's ready (again, see https://bugzilla.mozilla.org/show_bug.cgi?id=1174423)

**Comment by EmiXLabs** [comment]
@flodolo I'm following that bug as I'm in Cc and as you mentioned in Bugzilla, need to replicate what we did for Turkish locale also in Azerbaijani, just adding same CSS changes will be OK.

**Comment by kyoshino** [comment]
Sorry I'm late here :bow: Updated the PR based on the [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1174423)'s discussion. @flodolo r?

File: media/css/l10n/ga-IE/intl.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
-/* Bug 854756 */
+/* Bug 1174423 */
 
 * {
-  text-transform: none !important;
```
Comment (flodolo) [inline media/css/l10n/ga-IE/intl.css]: Let's keep this, we didn't receive any explicit request to remove it, and I don't know how bad it fails across browsers and versions.

**Comment by flodolo** [comment]
Besides one comment, can you add 'az' with the same rules as 'tr'? Then we're good to go.

**Comment by kyoshino** [comment]
Thanks @flodolo! Updated.

**Comment by flodolo** [comment]
r+ from me, thanks Kohei :+1:

**Pull request merged**